operator/v1/types_image_content_source_policy.go

```diff
@@ -0,0 +1,99 @@
+	//
+	// +optional
+	// +listType=map
+	// +listMapKey=source
+	RepositoryDigestMirrors []RepositoryDigestMirrors `json:"repositoryDigestMirrors"`
+}
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ImageContentSourcePolicyList lists the items in the ImageContentSourcePolicy CRD.
+//
+// Compatibility level 1: Stable within a major release for a minimum of 12 months or 3 minor releases (whichever is longer).
+// +openshift:compatibility-gen:level=1
+type ImageContentSourcePolicyList struct {
+	metav1.TypeMeta `json:",inline"`
+	metav1.ListMeta `json:"metadata"`
+
+	Items []ImageContentSourcePolicy `json:"items"`
+}
+
+// RepositoryDigestMirrors holds cluster-wide information about how to handle mirrors in the registries config.
+type RepositoryDigestMirrors struct {
+	// source is the repository that users refer to, e.g. in image pull specifications.
+	// see https://github.com/containers/image/blob/main/docs/containers-registries.conf.5.md#choosing-a-registry-toml-table for format specifications
+	// i.e. using the following format
+	// host[:port]
+	// host[:port]/namespace[/namespace…]
+	// host[:port]/namespace[/namespace…]/repo
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:Pattern=`^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\-]*[a-zA-Z0-9])\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\-]*[A-Za-z0-9])(:[0-9]+)?(\/[^\/]+)*$`
+	// +required
+	Source string `json:"source"`
+	// If true, the mirrors only work when pulling the images that are referenced by their digests.
+	// Pulling images by tag can potentially yield different images, depending on which endpoint
+	// we pull from. Forcing digest-pulls for mirrors avoids that issue.
+	// +optional
+	AllowMirrorByTags bool `json:"allowMirrorByTags,omitempty"`
+	// mirrors is zero or many repositories that may also contain the same images.
+	// A zero length mirrors or undefined mirrors field will not set a mirror, the image will continue to be pulled from the specified
+	// repository in the pull spec.
+	// For format of each mirror specifications, see https://github.com/containers/image/blob/main/docs/containers-registries.conf.5.md#choosing-a-registry-toml-table
+	// i.e. using the following format
+	// host[:port]
+	// host[:port]/namespace[/namespace…]
+	// host[:port]/namespace[/namespace…]/repo
+	// an empty string is not a valid mirror
+	// The order of mirrors in this list is treated as the user's desired priority, while source
+	// is by default considered lower priority than all mirrors. Other cluster configuration,
+	// including (but not limited to) other repositoryDigestMirrors objects,
+	// may impact the exact order mirrors are contacted in, or some mirrors may be contacted
+	// in parallel, so this should be considered a preference rather than a guarantee of ordering.
+	// +optional
+	// +listType=set
+	Mirrors []string `json:"mirrors,omitempty"`
```

Comment:
no format?

Reply:
`mirrors` is an already existed field of v1alpha1. Adding validation or change the current field is not planned for this PR.